core/queryparser/sparql/src/main/java/org/eclipse/rdf4j/query/parser/sparql/ast/ParseException.java

```diff
@@ -1,12 +1,5 @@
-/*******************************************************************************
- * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Distribution License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/org/documents/edl-v10.php.
- *******************************************************************************/
-/* Generated By:JavaCC: Do not edit this line. ParseException.java Version 5.0 */
-/* JavaCCOptions:KEEP_LINE_COL=null */
+/* Generated By:JavaCC: Do not edit this line. ParseException.java Version 7.0 */
+/* JavaCCOptions:KEEP_LINE_COLUMN=true */
 package org.eclipse.rdf4j.query.parser.sparql.ast;
```

Comment:
Fwiw I'm not particulary fussed about having the copyright headers on these generated files.